ExchangeSharp/API/Exchanges/ExchangeAPI.cs

```diff
@@ -132,6 +132,7 @@ public abstract class ExchangeAPI : BaseAPI, IExchangeAPI
         protected virtual IDisposable OnGetTickersWebSocket(Action<IReadOnlyCollection<KeyValuePair<string, ExchangeTicker>>> tickers) => throw new NotImplementedException();
         protected virtual IDisposable OnGetTradesWebSocket(Action<KeyValuePair<string, ExchangeTrade>> callback, params string[] symbols) => throw new NotImplementedException();
         protected virtual IDisposable OnGetOrderBookWebSocket(Action<ExchangeSequencedWebsocketMessage<KeyValuePair<string, ExchangeOrderBook>>> callback, int maxCount = 20, params string[] symbols) => throw new NotImplementedException();
+        protected virtual IDisposable OnGetOrderBooksDeltaSocket(Action<ExchangeSequencedWebsocketMessage<KeyValuePair<string, ExchangeOrderBook>>> action, params string[] symbols) => throw new NotImplementedException();
         protected virtual IDisposable OnGetCompletedOrderDetailsWebSocket(Action<ExchangeOrderResult> callback) => throw new NotImplementedException();
```

Comment:
I had considered that. We could certainly do that. I am OK with breaking API as this library is still very young and in "alpha" version. How much work would it save you if it was in the ticker and order book classes?

Reply:
Let's hold off for now. This is a hold-over from before web sockets last year when I had just implemented the GetTicker endpoint for a few exchanges. Since the symbol is always a parameter to that API call, I didn't add it to the model class. Same for order book. But in the future I may refactor to make it part of those classes.